styles.css

```diff
@@ -672,3 +714,638 @@ button:hover {
+.cross {
+  border: 1px solid #dfe1e6;
+  margin-top: 100px;
+  align-self: stretch;
+  background: #dfe1e6;
+  margin-left: 20px;
+}
+
+.cross:hover {
+  background: #172b4d;
+  border: 1px solid #172b4d;
+}
+
+footer {
+  bottom: 0;
+
+  /* display: block; */
+}
+
+.contact-social-icon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 20px;
+}
+
+.contact-social {
+  width: 375px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  margin-left: 20px;
+  padding-top: 40px;
+  margin-top: 80px;
+  bottom: 0;
+  align-self: center;
+  position: relative;
+  top: 2810px;
+  gap: 15px;
+}
+
+.line {
+  position: absolute;
+  width: 134px;
+  height: 5px;
+  left: calc(50% - 134px / 2 - 0.5px);
+  bottom: 20px;
+  background: #0d151e;
+  border-radius: 10px;
+  align-items: center;
+  margin-left: 10px;
+
+  /* margin-top: 20px; */
+}
+
+.desktop {
+  display: none;
+}
+
+@media (min-width: 600px) {
+  body {
+    margin: auto;
+  }
+
+  header,
+  section {
+    justify-content: center;
+    align-items: center;
+    min-width: (90%, 500px);
+  }
+
+  .AppBar {
+    height: 72px;
+    width: 100%;
+  }
+
+  .ToolBar {
+    width: 100%;
+  }
+
+  .headline {
+    width: 100%;
+    display: flex;
+    gap: 1rem;
+    height: 850px;
+  }
+
+  .Works {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-left: 5%;
+    margin-right: 5%;
+  }
+
+  .RecentWorks {
+    flex-direction: row;
+    width: 1300px;
+  }
+
+  .RecentWorks h2 {
+    width: 40%;
+    text-align: center;
+
+    /* margin-left: 2rem; */
+  }
+
+  #line {
+    width: 60%;
+    margin-left: -78px;
+  }
+
+  .MultiPostStories {
+    flex-direction: row;
+    width: 100%;
+    margin-top: 90px;
+    margin-bottom: 20px;
+  }
+
+  .MultiPostStories img {
+    width: 684px;
+    height: 448px;
+  }
+}
+
+@media (min-width: 875px) {
+  .NormalButton {
+    display: none;
+  }
+
+  /* .phone {
+    display: none;
+  } */
+  header,
+  section {
+    justify-content: center;
+    align-items: center;
+    min-width: (90%, 875px);
+  }
+
+  .AppBar {
+    height: 72px;
+    width: 100%;
+  }
+
+  .ToolBar {
+    width: 100%;
+  }
+
+  .nav-menu {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    gap: 100px;
+    margin-right: 120px;
+    margin-top: -20px;
+  }
+
+  .Logo {
+    margin-left: 120px;
+  }
+
+  .nav-menu li a {
+    align-self: flex-end;
+  }
+
+  .headline {
+    width: 100%;
+    display: flex;
+    gap: 1rem;
+    height: 850px;
+  }
+
+  .image-headline {
+    background: url("images/Illustration-header.png");
+    background-position: right;
+    background-size: cover;
+    background-repeat: no repeat;
+    width: 1000.13px;
+    height: 769.27px;
+    top: -140px;
+  }
+
+  .content {
+    display: flex;
+    flex-direction: row-reverse;
+    padding-top: 250px;
+  }
+
```

Comment:
- I think it will be better if we reduce the excess padding added to your `.content` style. Currently i believe the space at the top and bottom o this section is a bit too much. 🤗 

Here is what I mean 👇🏽 

## Your design
![faikeexce](https://user-images.githubusercontent.com/99246359/200112493-a9ef3794-b4e6-4ccc-825e-155afdbb7e16.PNG)

## Figma

![dieqfig](https://user-images.githubusercontent.com/99246359/200112568-3334f10a-3d84-483c-b6af-eccca68753d4.PNG)

How about we start first by reducing the `padding-top: 250px;`  to `padding-top: 100px;` then search for anywhere else you may have added some excess bottom padding or margin 🥂

Reply:
thanks for elaborated review and recommendation is on spot consider it done as you suggested

Reply:
Sure @Robison-faike123 , you are welcome 🤗